frontend/src/pages/Post/PostManager.tsx

```diff
@@ -7,26 +7,76 @@ import ImagePreviewList from '../../components/Post/ImagePreviewList';
 import { UserInfoContext } from '../../context/UserInfoContext';
 import useQuery from '../../hooks/useQuery';
 import { remote } from '../../lib/api';
-import { useNavigate } from '../../lib/Router';
+import memoryCache from '../../lib/MemoryCache';
+import { useNavigate, useSearchParams } from '../../lib/Router';
 import colors from '../../styles/colors';
 import { textSmall, textMedium } from '../../styles/fonts';
 import { CategoryType } from '../../types/category';
+import { ProductDetail } from '../../types/product';
+import { parseLocaleStringToNumber } from '../../utils/parse';
+
+interface ProductInputsType {
+  title: string;
+  description: string;
+  selectedCategory: number;
+  price: number;
+  thumbnails: string[];
+}
+
+const MAX_PRICE = 1000000000;
```

Comment:
전 이 단위의 크기를 읽기 어렵네요 ㅎ 어떤방법이 좋으려나..

Reply:
```suggestion
const MAX_PRICE = 1_000_000_000;
```

타입스크립트에서 이런 식으로 [numeric seperator](https://mariusschulz.com/blog/numeric-separators-in-typescript)를 지원한다고 하네요!
이러면 10억인걸 금방 알 수 있겠네요